### DISCUSSION TITLE
The kiai time star burst effect is missing

### DISCUSSION THREAD

**Discussion by smileyhead** [ppy/osu#20081] [Ideas] [answered]
osu!stable has a star burst effect for when kiai time starts, which, in my opinion, really improves the experience while playing a song. Lazer is currently missing this effect. Would it be possible to add it back?

https://user-images.githubusercontent.com/15932372/188006472-3ca74766-9d6b-4f59-9e45-11eacaf1b61d.mp4

**Comment by peppy** [comment, marked as answer]
Yep, definitely needs to be added back. These kinds of details are the polish that we haven't got to applying to lazer yet.

**Reply by peppy** [reply]
Tracking via https://github.com/ppy/osu/issues/21091.

**Comment by BKVad1m** [comment]
https://www.youtube.com/watch?v=1iFHftUNMrE kiai time preview in osu!stable
